HealthMonitoring.SelfHost/Content/Scripts/homeController.js

```diff
@@ -23,29 +26,68 @@
         var configFrequency = getConfigUpdatingFrequency($location);
 
         $scope.displayHint = function (evnt) {
-            if ($scope.statLegend == null) {
+            if ($scope.statLegend === null) {
```

Comment:
Yes., There should not have a situation when statLegend is 'undefined'. If we do, then something is amiss